webapp/components/test-file-results.js

```diff
@@ -82,7 +82,8 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
       },
       rows: {
         type: Array,
-        computed: 'computeRows(resultsTable, onlyShowDifferences)'
+        computed: 'computeRows(resultsTable, onlyShowDifferences)',
+        notify: true,
```

Comment:
We are removing two-way data binding in Polymer, in preparation for migration to Lit. Instead, could you use addEventListener?  E.g. https://github.com/web-platform-tests/wpt.fyi/pull/3341/files

Reply:
I'll see if I can re-work some things to incorporate that 👍

Reply:
Event listeners have been added